packages/components/Tabs/src/Tabs.tsx

```diff
@@ -137,8 +178,8 @@ export const Tabs = compose<TabsType>({
   slots: {
     root: View,
     label: Text,
-    container: FocusZone,
-    stack: { slotType: View, filter: filterViewProps },
+    container: Platform.OS !== 'windows' ? FocusZone : React.Fragment,
```

Comment:
I'm curious if the non-focus zone implementation would "just work" for other platforms as well that have keyboard events as well, like macOS. 
Perhaps a better question: What is the functionality we change/lose in the windows version vs the UWP version?

Reply:
Hmm, the main difference between the windows version and the UWP version is that on windows the keyboard focus hops from header (tabsItem) to header, while on the UWP version the focus stays on the stack when keyboarding.

Reply:
I see. That feels like a side effect of not having FocusZone. I would still add a comment with a GitHub issue number tracking adding FocusZone for windows.